docs/moov_stdlib/global.html

```diff
@@ -246,6 +246,64 @@ <h3 class="subsection-title">Members</h3>
     
     <dt class="tag-source">Source:</dt>
     <dd class="tag-source"><ul class="dummy"><li>
+        <a href="stdlib_js">stdlib_js</a>, <a href="stdlib_js#line24">line 24</a>
+    </li></ul></dd>
+    
+
+    
+
+    
+
+    
+</dl>
+
+
+
+
+
+
+        
+            
+
+
+
+
+
+
+
+
+
+
+
+<dl class="details">
+
+    
+
+    
+
+    
+
+    
+
+    
+
+    
+
```

Comment:
@cin210 Looks like compiling the extra whitespace has sort of always been a part of the framework: https://github.com/moovweb/moov_docs/blob/c1e90bc9afee3a379b92c0a7dd61d2c5146ef609/docs/moov_cheerio/global.html